src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityModule.cs

```diff
@@ -40,6 +41,8 @@ public class ParityModule : IParityModule
         private readonly IEnode _enode;
         private readonly ISignerStore _signerStore;
         private readonly IKeyStore _keyStore;
+        private readonly IPeerManager _peerManager;
+        private ParityNetPeers _parityNetPeers;
```

Comment:
_parityNetPeers shoudln't be a field in class. It can be a variable in parity_netPeers method.